### PR TITLE
fix a mistake

### PR DESCRIPTION
**Pull request by qingsenLi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind design

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/util.go [hunk 1]
```diff
@@ -4847,7 +4847,7 @@ func getMaster(c clientset.Interface) Address {
 		Failf("Failed to get kubernetes endpoints: %v", err)
 	}
 	if len(eps.Subsets) != 1 || len(eps.Subsets[0].Addresses) != 1 {
-		Failf("There are more than 1 endpoints for kubernetes service: %+v", eps)
+		Failf("There is more than 1 endpoint for kubernetes service: %+v", eps)
```
Comment (neolit123) [inline test/e2e/framework/util.go]: both the new and old forms are grammatically correct.
please leave the original.

File: test/e2e_node/e2e_node_suite_test.go [hunk 2]
```diff
@@ -277,7 +277,7 @@ func getNode(c *clientset.Clientset) (*v1.Node, error) {
 	if nodes == nil {
 		return nil, fmt.Errorf("the node list is nil.")
 	}
-	Expect(len(nodes.Items) > 1).NotTo(BeTrue(), "should not be more than 1 nodes.")
+	Expect(len(nodes.Items) > 1).NotTo(BeTrue(), "should not be more than 1 node.")
```
Comment (neolit123) [inline test/e2e_node/e2e_node_suite_test.go]: this is a valid change, but instead we should use:
> the number of nodes is more than 1

**Review by neolit123 — COMMENTED** [review]
@qingsenLi i've added a couple of comments.

please squash / force push the commits so that we have one for this PR.
thanks.

/kind cleanup
/priority backlog

**Comment by qingsenLi** [comment]
/cc @neolit123 
thanks for review.

**Review by neolit123 — COMMENTED** [review]
thanks for the update
/lgtm

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**